react-charts/bullet-chart/step5.md

```diff
@@ -1,43 +1,73 @@
-Now we'll add labels to the bullet chart...
+Add a legend to the bullet chart.
 
-## Task
+1. <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2. <strong>Locate the `<ChartBullet/>` component:</strong>
 
-2) Locate the code that looks like the following:
+I will look like this:
 
 <pre class="file">
 &lt;ChartBullet
-  comparativeWarningMeasureData={[{ name: &#39;Warning&#39;, y: 88 }]}
-  maxDomain={{y: 100}}
   primarySegmentedMeasureData={[{ name: &#39;Measure&#39;, y: 60 }]}
   qualitativeRangeData={[{ name: &#39;Range&#39;, y: 50 }, { name: &#39;Range&#39;, y: 75 }]}
+  comparativeWarningMeasureData={[{ name: &#39;Warning&#39;, y: 88 }]}
+  maxDomain={{y: 100}}
   height={150}
   width={600}
+  constraingToVisibleArea
+  labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
+  padding={{
+    bottom: 50,
+    left: 150,
+    right: 50,
+    top: 50
+  }}
+  title="Text label"
+  subTitle="Measure details"
 /&gt;
 </pre>
 
-3) Add the following property to that section:
+3. <strong>Add the `legendOrientation` property to the `ChartBullet` component:</strong>
+
+You will need to set the orientation so that the legend behaves as expected when you add data to it in the final steps (there will be no visible changes yet).

```

Comment:
```suggestion
The orientation is set so that the legend behaves as expected when you add data to it in the final steps (there will be no visible changes yet).
```

Reply:
```suggestion
Legend orientation is set so that it behaves as expected when you add legend data in the final steps (there will be no visible changes yet).
```